City-of-Helsinki/drupal-hdbt
# UHF-2828: siteimprove & matomo javascript

Adds Siteimprove & Matomo tracking codes to the theme.

How to test:

- Set up any of the sites
  - Note that here should be `$settings['matomo_site_id']` and `$settings['siteimprove_id']` set from env variables in `settings.php` (there are site specific pull requests for those, see the mentions below), you can also set them manually if they don't exist (actual values provided in Slack per request)
- Checkout this branch of the theme: `composer require drupal/hdbt:dev-UHF-2828_matomo_siteimprove_javascript`
- Clear cache: `make drush-cr`
- Go to any content page and
  - View page source and search for "siteimprove", there should be a `<script>` element containing that at the end of the page
  - Inspect network traffic and search for "digiaiiris", there should be two items
